packages/contracts/contracts/randomizer/shared/SharedRandomizerV0.sol

```diff
@@ -180,7 +172,7 @@ contract SharedRandomizerV0 is ISharedRandomizerV0 {
         // does not require any gating.
```

Comment:
Unrelated to this PR specifically, and can be separate follow up, but I think we probably want to update the documentation to more aggressively clarify that there are specific assumptions around this method being used atomically at the time of mint by the core contract that shares a relationship to the randomizer implementation.

Reply:
sounds good - very much agree this is communicated very weirdly 😅 will toss in a follow-up PR 🙏